bothub/utils.py

```diff
@@ -324,3 +351,54 @@ class CountSubquery(Subquery):
 
     def __init__(self, queryset, output_field=None, **extra):
         super().__init__(queryset, output_field, **extra)
+
+
+class AbstractService:
```

Comment:
does it make sense to call yourself "Abstract Service"? since it implements specific methods? Perhaps it would be necessary to change the nomenclature.

Why is it called "Abstract" if it doesn't have any abstract methods?